assignment_A2.ipynb

```diff
@@ -1321,16 +1321,16 @@
     "        \n",
     "        return item\n",
     "\n",
-    "    def in_order(self, child, parent): #SHOULD WE COMPARE THE VALUES OR THE VALUES AT THESE ENTREES? \n",
+    "    def in_order(self, child, parent): #We should compare values!\n",
     "        '''return T|F parent and child are in heap-order'''\n",
     "        if parent is None or child is None:\n",
     "            return True\n",
     "        \n",
     "        #MIN_HEAP, MAX_HEAP = 0, 1\n",
-    "        if self.hp_type: #MAX_HEAP\n",
-    "            return child <= parent\n",
+    "        if self.type: #MAX_HEAP\n",
+    "            return self[parent] >= self[child]\n",
     "        else: #MIN_HEAP\n",
-    "            return parent >= child \n",
+    "            return self[parent] <= self[child] \n",
```

Comment:
We should compare values rather than indices (duh). Furthermore fixed sloppy comparison.